ProcessEngineClient/ProcessEngineClient.cs

```diff
@@ -0,0 +1,123 @@
+namespace ProcessEngineClient
+{
+    using System;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Threading.Tasks;
+
+    using ProcessEngine.ConsumerAPI.Client;
+    using ProcessEngine.ConsumerAPI.Contracts;
+    using ProcessEngine.ConsumerAPI.Contracts.DataModel;
+
+    using ProcessEngine.ExternalTaskAPI.Client;
+    using ProcessEngine.ExternalTaskAPI.Contracts;
+
+
+    public class ProcessEngineClient 
+    {
+        private HttpClient HttpClient { get; }
+
+        private Identity Identity {get; set;}
+
+        private ConsumerApiClientService ConsumerApiClient { get; }
+
+        private IExternalTaskAPI ExternalTaskApi { get; }
+
+
+        public ProcessEngineClient(string url) 
+            : this(url, Identity.DefaultIdentity)
+        {   
+        }
+
+        public ProcessEngineClient(string url, Identity identity) 
+        {
+            this.HttpClient = new HttpClient();
+            this.HttpClient.BaseAddress = new Uri(url);
+            this.Identity = identity;
+
+            this.ConsumerApiClient = new ConsumerApiClientService(this.HttpClient);
+
+            this.ExternalTaskApi = new ExternalTaskApiClientService(this.HttpClient);
+        }
+        public async Task<ProcessStartResponse<object>> StartProcessInstance(
```

Comment:
Wenn ich mir die beiden unteren Signaturen ansehe, wirkt diese hier eher überflüssig. 

Falls nicht, Spacing nach oben hin fehlt :wink:  

```suggestion

        public async Task<ProcessStartResponse<object>> StartProcessInstance(
```